EnglishTranslation-master/ETC.tsv

```diff
@@ -28777,7 +28777,7 @@ ETC_20170912_028771	/whale	/고래사냥
 ETC_20170912_028772	Opening this cube consumes {CONSUME}.{nl}Are you sure you want to open it?	큐브 개봉시{nl}{CONSUME}{nl}소모됩니다.{nl}정말 개봉하시겠습니까?				
 ETC_20170912_028773	Cannot be used near Goddess Statues.	여신상 주변에서는 사용하실 수 없습니다.				
 ETC_20170912_028774	Cannot be used near boss monsters.	보스 몬스터 주변에서는 사용하실 수 없습니다.				
-ETC_20170912_028775	Can only be used in fields.	필드에서만 사용하실 수 있습니다.				
+ETC_20170912_028775	You can only use this in fields.	필드에서만 사용하실 수 있습니다.				
```

Comment:
How about, `This can only be used in fields.` ?

The same could be done to `ETC_20170912_028774` and `ETC_20170912_028773`
i.e `ETC_20170912_028774	This cannot be used near boss monsters.`